### DISCUSSION TITLE
希望夜间模式有更黑的背景，提高对比度

### DISCUSSION THREAD

**Discussion by XPRAMT** [the1812/Bilibili-Evolved#4401] [功能建议 / Ideas]
纯黑背景会比现在的看起来更舒服一点，像下面这样
<a href="https://www.z4a.net/image/Vvtllw"><img src="https://www.z4a.net/images/2023/09/01/-2023-09-01-113725.png" alt="-2023-09-01-113725.png" border="0"></a>
<a href="https://www.z4a.net/image/VvtzWN"><img src="https://www.z4a.net/images/2023/09/01/-2023-09-01-113750.png" alt="-2023-09-01-113750.png" border="0"></a>

**Comment by Geeyun-JY3** [comment]
#3820
